RELEASE_NOTES

```diff
@@ -1,3 +1,9 @@
+========================================================================
+Amazon CloudWatch Agent Operator v1.3.1 (2024-04-24)
+========================================================================
+Enhancements:
+* [Windows-ContainerInsights] Fix CI Windows issue with containerd 1.7
```

Comment:
Should this be a little more descriptive?

"Pass WorkingDir to agent container if defined in the AmazonCloudWatchAgent spec to fix issue with containerd 1.7 on windows"?

Reply:
Updated !